src-tauri/src/northstar/nslog.rs

```diff
@@ -0,0 +1,34 @@
+use crate::{NorthstarLog, GameInstall};
+
+#[tauri::command]
+pub fn fetch_northstar_logs(
+    game_install: GameInstall,
+) -> Result<Vec<NorthstarLog>, String> {
+
+    let mut northstar_logs: Vec<NorthstarLog> = Vec::new();
+
+    let pattern = format!("{}/R2Northstar/logs/nslog*.txt", game_install.game_path);
+    dbg!(&pattern);
+    for e in glob::glob(&pattern).expect("Failed to read glob pattern") {
+        let file = e.unwrap();
+
+        let log = NorthstarLog {
+            filename: file.file_name().unwrap().to_os_string().into_string().unwrap(),
+            path: file.to_str().unwrap().to_string(),
```

Comment:
same thing

Reply:
I am not quite sure what unwrap you want me to remove here since the `NorthstarLog` struct takes `String` not some `Option<>`

Reply:
map it to a error then map it to a string